src/cliHelper.cpp

```diff
@@ -1,40 +1,9 @@
-#include "utilities.hpp"
+#include "cliHelper.hpp"
 
-////////////////////////
-// CLOCK
```

Comment:
note: I moved MillisecondClock, cliHelper, and myRandom to their own respective files. These types had disparate dependencies and were also used in different places, so it didn't make sense to group them together. Also as a general best practice, it's best to have one class per file.

See also other comment on utilities.hpp